<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partSettingsColors</class>
 <widget class="QWidget" name="partSettingsColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>132</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1,2">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblVisualizerBackgroundColor">
       <property name="text">
        <string>Background:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorPicker" name="clpVisualizerBackgroundColor" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="ColorPicker" name="clpVisualizerTableGridColor" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Table grid:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorPicker" name="clpVisualizerToolColor" native="true"/>
     </item>
     <item row="0" column="3">
      <widget class="ColorPicker" name="clpVisualizerTextColor" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Tool:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblVisualizerToolpathColors">
     <property name="font">
      <font>
       <italic>false</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Toolpath:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_13" columnstretch="1,2,1,2">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Highlight:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblVisualizerZMovementColor">
       <property name="text">
        <string>Z-movement:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorPicker" name="clpToolpathNormalColor" native="true"/>
     </item>
     <item row="1" column="3">
      <widget class="ColorPicker" name="clpToolpathZMovementColor" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="ColorPicker" name="clpToolpathRapidMovementColor" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="ColorPicker" name="clpToolpathHighlightColor" native="true"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>End point:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="ColorPicker" name="clpToolpathEndColor" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblVisualizerNormalColor">
       <property name="text">
        <string>Normal:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rapid movement:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Drawn:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ColorPicker" name="clpToolpathStartColor" native="true"/>
     </item>
     <item row="0" column="3">
      <widget class="ColorPicker" name="clpToolpathDrawnColor" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Start point:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
